src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,42 @@
+@use '../utils/variables' as *;
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+  width: 100%;
+  padding: 0 16px;
+  box-sizing: border-box;
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    width: 96px;
+    justify-content: space-around;
+  }
+
+  &__star {
+    background-image: url(./images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__stars--4
+  &__star:nth-child(-n + 4) {
```

Comment:
Consider keeping modifiers working

Reply:
I don't get it ;]
It works just fine to me...

Reply:
<img width="844" alt="Screenshot 2024-01-30 at 16 36 52" src="https://github.com/mate-academy/layout_catalog/assets/135232001/67edba42-eae0-498a-a263-48fa671dacff">
The stars structure and modifiers should work as in the previous task

Reply:
Ok, I get it now. Fixed ;]